### ISSUE TITLE
[FEAT] How can i set min and max bounds for columns with numberic data?

### ISSUE THREAD

**Issue by tsiakmaki** [ydataai/ydata-synthetic#328]
Our columns with numeric data are integers above or equal to zero. We noticed that the output synthetic data contains numbers below zero in those columns.
Is there any way to set thresholds for the synthetic numeric data?

Thank you in advance for your effort.

**Comment by AryanSaeedi** [comment]
Yeah, I am getting the same issue with all of the models I worked with so far. DRAGAN, WGAN, WGANGP, WGAN, and with CramerGAN after a while I get _nan_ for generator's and discriminator's losses. I am using a network traffic dataset and it can't have negative values. I even tried running WGANGP for 3000 epochs thinking it might try to improve itself but still, I was getting negative values. My dataset is (300000, 80) where only one column is categorical and the rest are continuous. 
Please let me know if you were able to solve the issue. 

Thank you
